### PR TITLE
Bugfix: invalid path to run.gdb file

### PR DESCRIPTION
**Pull request by andreas-bok-sociomantic**
Fixes issue where `make gdb-*` failed
due to invalid path to run.gdb

### CODE REVIEW

File: yatl/include.am [hunk 1]
```diff
@@ -17,8 +17,7 @@ export LIBTOOL_COMMAND
 export VALGRIND_COMMAND
 export HELGRIND_COMMAND
 export DRD_COMMAND
-export GDB_COMMAND 
-  
+export GDB_COMMAND
```
Comment (ben-palmer-sociomantic) [inline yatl/include.am]: Is this change related to this commit?

Comment (andreas-bok-sociomantic) [reply yatl/include.am]: No, newline sneaked in.

**Comment by andreas-bok-sociomantic** [comment]
Reverted unrelated change

File: yatl/include.am [hunk 2]
```diff
@@ -18,7 +18,6 @@ export VALGRIND_COMMAND
 export HELGRIND_COMMAND
 export DRD_COMMAND
 export GDB_COMMAND 
-
```
Comment (ben-palmer-sociomantic) [inline yatl/include.am]: Its sneaked back in again :)

Comment (andreas-bok-sociomantic) [reply yatl/include.am]: > Its sneaked back in again :)

Damn it. :) Fix now

**Comment by ben-palmer-sociomantic** [comment]
LGTM
